featuretools/demo/mock_customer.py

```diff
@@ -27,7 +33,7 @@ def load_mock_customer(n_customers=5, n_products=5, n_sessions=35, n_transaction
     transactions_df = transactions_df.sort_values("session_id").reset_index(drop=True)
     transactions_df["transaction_time"] = pd.date_range('1/1/2014', periods=n_transactions, freq='65s')  # todo make these less regular
     transactions_df["product_id"] = pd.Categorical(choice(products_df["product_id"], n_transactions))
-    transactions_df["amount"] = random.randint(500, 15000, n_transactions) / 100.0
+    transactions_df["amount"] = old_div(random.randint(500, 15000, n_transactions), 100.0)
```

Comment:
was the swapping in of `old_div` automatic? In many of these cases, I think we actually can just use the true divide with python 3 and `from __future__ import division` to support python 2

Reply:
Sounds good, this was automatically applied during the conversion. Just mark anything that doesn't need it and i'll switch it over :)